pyrmq/__init__.py

```diff
@@ -8,12 +8,12 @@
 """
 from pkg_resources import get_distribution
 
-from pyrmq.consumer import Consumer
 from pyrmq.publisher import Publisher
+from pyrmq.consumer import Consumer
```

Comment:
Any reason why we changed order of imports?

Reply:
Yes. Consumer now needs Publisher to be fully imported because it uses it.

Reply:
Oh, that's not good. Maybe you can avoid this by moving the import inside the function where it is used in the Consumer class.